app/App.tsx

```diff
@@ -36,16 +39,16 @@ const getStyle = (index: number, showLabels: boolean):StyleSpecification => {
     sources: {
       protomaps: {
         type: "vector",
-        url: "https://api.protomaps.com/tiles/v4.json?key=1003762824b9687f",
```

Comment:
Can you revert this? The `1003762824b9687f` key works on all *github.io subdomains and localhost:5173